.github/workflows/acceptance-tests.yml

```diff
@@ -121,17 +130,16 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: ${{ env.terraform_version }}
+          terraform_wrapper: false  
       - name: Acceptance Tests
         env:
           MONGODB_ATLAS_PUBLIC_KEY: ${{ secrets.MONGODB_ATLAS_PUBLIC_KEY_CLOUD_DEV }}
           MONGODB_ATLAS_PRIVATE_KEY: ${{ secrets.MONGODB_ATLAS_PRIVATE_KEY_CLOUD_DEV }}
           MONGODB_ATLAS_ORG_ID: ${{ vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV }}
           MONGODB_ATLAS_BASE_URL: ${{ vars.MONGODB_ATLAS_BASE_URL }}
-          ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
```

Comment:
no need to provide ACCTEST_TIMEOUT, PARALLEL_GO_TEST, SKIP_TEST_EXTERNAL_CREDENTIALS if they're using default values

Reply:
Right now we can change the timeout by setting its value as var in the secret section, if you remove it we won't be able to do it anymore. Just flagging this behaviour since sometimes we use it

Reply:
ah, ok, I thought we never used it, ok I can keep that in the common vars in case we want to do that.  
(in a PR you can always change temporarily this file but i can use the var, that's fine)